core/namespaces/namespaces-tests.factor

```diff
@@ -38,3 +38,29 @@ SYMBOL: toggle-test
 
 { t } [ [ test-initialize get-global ] { at* set-at } inlined? ] unit-test
 { t } [ [ test-initialize set-global ] { at* set-at } inlined? ] unit-test
+
+{ t } [
+    f toggle-test set
+    toggle-test [ not ] change
+    toggle-test get
+] unit-test
+
+SYMBOL: counter
```

Comment:
probably would change this so not to shadow the ``namespaces:counter`` word.

Reply:
renamed